pkg/controller/storagecluster/cephcluster.go

```diff
@@ -510,3 +517,10 @@ func (r *ReconcileStorageCluster) throttleStorageDevices(storageClassName string
 
 	return false, false, nil
 }
+
+func (r *ReconcileStorageCluster) tuneDeviceType(deviceType string) (bool, bool) {
```

Comment:
IMO we should change the name of this func as we are not tuning anything here.

Reply:
done